python/lsst/cp/verify/verifyFlat.py

```diff
@@ -0,0 +1,199 @@
+                         pipelineConnections=CpVerifyStatsConnections):
+    """Inherits from base CpVerifyStatsConfig.
+    """
+
+    def setDefaults(self):
+        super().setDefaults()
+        self.imageStatKeywords = {'MEAN': 'MEAN',  # noqa F841
+                                  'NOISE': 'STDEVCLIP', }
+        self.detectorStatKeywords = {'MEAN': 'MEAN',  # noqa F841
+                                     'SCATTER': 'STDEV', }
+
+
+class CpVerifyFlatTask(CpVerifyStatsTask):
+    """Flat verification sub-class, implementing the verify method.
+    """
+    ConfigClass = CpVerifyFlatConfig
+    _DefaultName = 'cpVerifyFlat'
+
+    def detectorStatistics(self, statisticsDict, statControl):
+        """Calculate detector level statistics based on the existing
+        per-amplifier measurements.
+
+        Parameters
+        ----------
+        statisticsDictionary : `dict` [`str`, `dict` [`str`, scalar]],
+            Dictionary of measured statistics.  The inner dictionary
+            should have keys that are statistic names (`str`) with
+            values that are some sort of scalar (`int` or `float` are
+            the mostly likely types).
+
+        Returns
+        -------
+        outputStatistics : `dict` [`str`, scalar]
+            A dictionary of the statistics measured and their values.
+
+        """
+        outputStatistics = {}
+
+        ampStats = statisticsDict['AMP']
+        amplifierMeans = [stats['MEAN'] for stats in ampStats.values()]
+
+        statisticToRun, statAccessor = self._configHelper(self.config.detectorStatKeywords)
+        stats = afwMath.makeStatistics(amplifierMeans, statisticToRun, statControl)
+
+        for k, v in statAccessor.items():
+            outputStatistics[k] = stats.getValue(v)
+
+        return outputStatistics
+
+    def verify(self, exposure, statisticsDict):
+        """Verify that the measured statistics meet the verification criteria.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            The exposure the statistics are from.
+        statisticsDictionary : `dict` [`str`, `dict` [`str`, scalar]],
+            Dictionary of measured statistics.  The inner dictionary
+            should have keys that are statistic names (`str`) with
+            values that are some sort of scalar (`int` or `float` are
+            the mostly likely types).
+
+        Returns
+        -------
+        outputStatistics : `dict` [`str`, `dict` [`str`, `bool`]]
+            A dictionary indexed by the amplifier name, containing
+            dictionaries of the verification criteria.
+        success : `bool`
+            A boolean indicating if all tests have passed.
+        """
+        ampStats = statisticsDict['AMP']
+        verifyStats = {}
+        success = True
+        for ampName, stats in ampStats.items():
+            verify = {}
+
+            # DMTN-101 Test 10.X: confirm that per-amplifier scatter is
```

Comment:
10.X was my shorthand for "this will be defined in section 10, but isn't formally done yet."  I want to get the code written first, because adding/updating tests is easy once the code exists, and getting something running is easier than getting DMTN-101 updates approved.